### ISSUE TITLE
Beams are invisible behind or inside glass

### ISSUE THREAD

**Issue by mastan-rus** [sultim-t/xash-rt#76]
Video: https://youtu.be/vqwfoUn9ukQ

Tau Cannon impact also not visible behind glass, only the glow of its particles.

**Comment by sultim-t** [comment]
Yup, in this renderer, beams and portals sprites are not treated as ray traced geometry, as they're semitransparent. So they can't be seen behind glass (reflective / refractive surface)
